src/screens/Customers/EditCustomerTariffPlans.tsx

```diff
@@ -66,7 +68,7 @@ function EditCustomerTariffPlans(props: EditCustomerTariffPlansProps) {
   }
 
   return (
-    <BaseScreen title="Tariff Plans" leftIcon="backNav">
+    <BaseScreen title={intl(words.tariff_plan)} leftIcon="backNav">
       <BaseScrollView>
         <List>
           {availableTariffPlans.map((tariffPlan: TariffPlanRecord) =>
```

Comment:
kind of unrelated, but I saw in `TariffPlanCard` there are some untranslated `Ks` and `kWh`, would you mind also translating those in this PR?